src/_kw

```diff
@@ -0,0 +1,25 @@
+#compdef kw
```

Comment:
Is this file specific to zsh? If so, I recommend you add a small piece of documentation on top of this file that clarifies this file target.

Reply:
Good point! Added a small description explaining to the purpose of the file.